launcher/src/main/java/org/apache/spark/launcher/CommandBuilderUtils.java

```diff
@@ -334,6 +334,18 @@ static void addPermGenSizeOpt(List<String> cmd) {
   }
 
   /**
+   * Gets the OutOfMemoryError option for Spark if the user hasn't set it.
+   */
+  public static void addOutOfMemoryErrorArgument(List<String> cmd) {
```

Comment:
Just YarnSparkHadoopUtil ? it can be inlined the one place it's called or am I overlooking something?